wlauto/instrumentation/fps/__init__.py

```diff
@@ -221,23 +261,40 @@ def __init__(self, outfile, device, activities, keep_raw, logger, dumpsys_period
         self.frames = []
         self.last_ready_time = 0
         self.refresh_period = VSYNC_INTERVAL
-        self.drop_threshold = self.refresh_period * 1000
+        self.drop_threshold = self.refresh_period * 10
```

Comment:
What do you mean? it was used inside `_process_trace_line`. The *1000 multiplier is somewhat arbitrary, but it's set large enough so that we'll never see it in real frames -- this is used to catch "bogus" frames that are sometimes output by SurfaceFlinger. 

The threshold of *10 refresh_period translates to 6fps. While this is definitely low, it is conceivable, so this has the potential of dropping real frames.